NDTensors/Project.toml

```diff
@@ -37,3 +37,26 @@ Strided = "0.3, 1"
 TimerOutputs = "0.5.5"
 TupleTools = "1.2.0"
 julia = "1.6"
+
+[weakdeps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+Metal = "dde4c033-4e86-420c-a63e-0dd931031962"
+TBLIS = "48530278-0828-4a49-9772-0f3830dfa1e9"
+Octavian = "6fd5a793-0b7e-452c-907f-f8bfe9c57db4"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Is Zygote supposed to be a weakdep?

Reply:
No this was for testing in `ext/NDTensorCUDA/examples.jl`. This is outdated from when I was trying to add the dependencies need in `NDTensors/test/` . I removed all Zygote from this file.